### DISCUSSION TITLE
Request for Assistance with Extracting Data from AntiSMASH Output

### DISCUSSION THREAD

**Discussion by Timmy16** [antismash/antismash#746] [General]
Hello everyone,
I hope this message finds you well. I am fairly new to bioinformatics. I am currently exploring the AntiSMASH tool and am eager to understand its outputs better, particularly how to work with the data generated in the region.js and output.json files.
I am interested in creating a result table or a CSV file from the AntiSMASH output. I would greatly appreciate any guidance or resources on how to extract relevant information from these files effectively.
Could anyone please share their insights or suggestions on the best practices for achieving this? Additionally, if there are any existing tools or scripts that could facilitate this process, I would be eager to learn about them.
Thank you for your time and assistance! I look forward to your responses.

**Comment by zreitz** [comment]
The JSON contains all of the info from an antiSMASH run. Pretty much any language can parse a JSON, but python would probably be easiest for you. There have been a couple previous issue discussions that give examples and strategies if you are interested in writing your own python code: [link](https://github.com/antismash/antismash/issues/443), [link](https://github.com/antismash/antismash/issues/128). 

Alternatively, I've already written full scripts to generate TSVs from a collection of antiSMASH runs, as part of my antiSMASH wrapper [multiSMASH](https://github.com/zreitz/multismash). Example outputs are shown [here](https://github.com/zreitz/multismash#standalone-scripts).  If the TSV doesn't have the columns you want, feel free to adapt the scripts.

**Reply by Timmy16** [reply]
Thank you so much for the info. Appreciate it.
